src/control/cmd/dmg/pool.go

```diff
@@ -405,12 +405,24 @@ type PoolQueryCmd struct {
 	ctlInvokerCmd
 	jsonOutputCmd
 	UUID string `long:"pool" required:"1" description:"UUID of DAOS pool to query"`
+	Targetlist string `long:"tgtidx" description:"Comma-separated list of pool targets to be queried on the rank"`
```

Comment:
Remove "on the rank", confusing. All pool ranks are queried but only for specific pool targets